torch/distributed/_tensor/ops/_pointwise_ops.py

```diff
@@ -546,9 +546,15 @@ def linear_pointwise_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> Strategy
     aten._foreach_clamp_min_.Scalar,
     aten._foreach_div_.List,
```

Comment:
Should we also add aten. _foreach_div_.Scalar and aten. _foreach_div.Scalar in this PR as well?

Reply:
yep good call